src/app/pages/sharing/components/shares-dashboard/smb-card/smb-card.component.ts

```diff
@@ -82,6 +83,7 @@ export class SmbCardComponent implements OnInit {
           iconName: 'edit',
           tooltip: this.translate.instant('Edit'),
           onClick: (row) => this.openForm(row),
+          disabled: (row) => this.changeEnabledIds$.pipe(map((ids) => ids.has(row.id))),
```

Comment:
I think you misunderstood the issue. The author means `disabled=unchecked` and `enabled=checked`. The problem is that the state is `unsynced` for toggle&checkbox between card and form.